PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,92 @@
+### Description
+<!-- Add description of work done here -->
+
+### Spec
+
+Designs: [Designs](DESIGN_URL)
+
+See Story: [ISSUE_NUMBER]((https://sparkbox.atlassian.net/browse/FSA22V1-ISSUE_NUMBER))
+
+### Validation
+<!-- Delete anything irrelevant to this PR -->
+
+* [ ] This PR has visual elements, so it was reviewed by a designer.
+* [ ] This PR has code changes, and our linters still pass.
+* [ ] This PR affects production code, so it was browser tested (see below).
+* [ ] This PR has new code, so new tests were added or updated, and they pass.
+* [ ] This PR has new SCSS functions, mixins, or variables, so [Sass Tests](https://seesparkbox.com/foundry/how_and_why_we_unit_test_our_sass) were added or updated, and they pass.
+* [ ] This PR has copy changes, so copy was proofread and approved.
+* [ ] The content of this PR requires documentation, so we added a detailed description of the component's purpose, requirements, quirks, and instructions for use by designers and developers. Along with accessibility information if pertinent.
+
```

Comment:
In addition to removing the Browser Testing section from the PR template, I think we should have another item in the checklist for general browser testing.

```suggestion
* [ ] Changes are browser tested. This includes functionality, accessibility, responsiveness, and design.

```